hathor/wallet/wallet.py

```diff
@@ -17,9 +17,9 @@
 import os
 from typing import Any, Optional
 
-from cryptography.hazmat.backends.openssl.ec import _EllipticCurvePrivateKey
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey
```

Comment:
Same here: Is it still using OpenSSL?

Reply:
I removed this from this PR.

Reply:
Yes. Similarly to the other one, in this case we were importing a private class that inherits from `ec.EllipticCurvePrivateKey`. Check it out in the current version (`38.0.3`):

https://github.com/pyca/cryptography/blob/38.0.3/src/cryptography/hazmat/backends/openssl/ec.py#L125

And in the new version (`42.0.5`) we are just importing that same abstract base class, which is also registered with an OpenSSL backend:

https://github.com/pyca/cryptography/blob/42.0.5/src/cryptography/hazmat/primitives/asymmetric/ec.py#L125

Lastly, from the docs, check the [Use of OpenSSL](https://cryptography.io/en/42.0.5/openssl/) page. It's clear that OpenSSL is the only tool used for all cryptographic operations and that its backend is a private implementation detail.

Reply:
Great. Thanks!